pkg/controller/direct/securesourcemanager/mapper.go

```diff
@@ -58,3 +61,20 @@ func OperationMetadata_EndTime_ToProto(mapCtx *direct.MapContext, in *string) *t
 	mapCtx.Errorf("OperationMetadata_EndTime_ToProto not implemented")
 	return nil
 }
+
+func SecureSourceManagerRepositoryObservedState_FromProto(mapCtx *direct.MapContext, in *pb.Repository) *krm.SecureSourceManagerRepositoryObservedState {
```

Comment:
Do you know why this is not generated to mappers.generated.go?

Reply:
Not sure... I initially thought it is because the ObservedState struct wasn't populated with fields, but it does have one now..